zeppelin-web/src/app/notebook/paragraph/paragraph-control.html

```diff
@@ -26,6 +26,20 @@
   <span class="icon-control-play" style="cursor:pointer;color:#3071A9" tooltip-placement="top" tooltip="Run this paragraph (Shift+Enter)"
         ng-click="runParagraph(getEditorValue())"
         ng-show="paragraph.status!='RUNNING' && paragraph.status!='PENDING' && paragraph.config.enabled"></span>
+  <span ng-show="paragraph.runtimeInfos.jobUrl.length == 1">
+    <a href="{{paragraph.runtimeInfos.jobUrl[0]}}" target="_blank"><span class="fa fa-tasks"></span> Spark job </a>
```

Comment:
How about display runtime info button before paragraph status and run button?

![image](https://cloud.githubusercontent.com/assets/1540981/20610874/42a2ee96-b255-11e6-9d3d-db2492fdd4ad.png)

![image](https://cloud.githubusercontent.com/assets/1540981/20610875/4d0a8ed4-b255-11e6-8b8a-c4254006a719.png)

So all other other control buttons are stay in the same position whether runtimeInfo button is displayed or not.